Makefile

```diff
@@ -135,7 +136,9 @@ endif
 build: toolchain standalone $(TOP)/sdk sdk_patch $(TOOLCHAIN)/xtensa-lx106-elf/lib/libhal.a utils libcirom
 build-prebuilt-toolchain: standalone $(TOP)/sdk sdk_patch utils
 
-utils: esptool esptool2 memanalyzer
+
+utils:
+#utils: esptool esptool2 memanalyzer
```

Comment:
This is a little mistake, it should not be in the PR.
I uncommented utils, because I still do not need them ATM.